package.json

```diff
@@ -32,7 +32,8 @@
   "homepage": "https://github.com/hmrc/design-system#readme",
   "dependencies": {
     "govuk-frontend": "^2.11.0",
-    "hmrc-frontend": "^0.5.0"
+    "hmrc-frontend": "^0.5.0",
+    "iframe-resizer": "^4.1.1"
```

Comment:
I'd say this is a `devDependency` as it's static site generated ... `govuk-frontend` and `hmrc-frontend` could both be `devDependency` too but this sets them apart in a way that I find helpful.

Reply:
👍